client/src/components/PersonCard/PersonCard.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PersonCard from './PersonCard';
+import Image from "../../assets/images/profile_large.png";
+import { storiesOf } from '@storybook/react';
+
+storiesOf('PersonCard', module)
+.add("Default", () => <PersonCard
+  PersonCard__image type="large" image={Image} 
+  PersonCard__name="Lindsay L." 
+  PersonCard__location="Washington D.C." 
+  PersonCard__interests={[ 
+  { url: "#", interests: "veganism" },
+  { url: "#", interests: "budgeting" },
+  { url: "#", interests: "travel" },
+  { url: "#", interests: "outdoors"}
```

Comment:
Didn't you fix these prop names in a previous PR? These should just be `image`, `name`, `location`, etc - not `PersonCard__image`